common/rpc/interceptor/telemetry.go

```diff
@@ -235,52 +235,40 @@ func (ti *TelemetryInterceptor) handleError(
 
 	scope.Tagged(metrics.ServiceErrorTypeTag(err)).IncCounter(metrics.ServiceErrorWithType)
 
-	if common.IsContextDeadlineExceededErr(err) {
-		scope.IncCounter(metrics.ServiceErrContextTimeoutCounter)
```

Comment:
Shouldn't we remove the metrics definition as well from `common/metrics/defs.go`?

Reply:
Yes, we should. But apparently, some of them are used somewhere else.